File: surf-api-bukkit/surf-api-bukkit-api/src/main/java/dev/slne/surf/surfapi/bukkit/api/nms/listener/packets/serverbound/CommandSuggestionPacket.java
package dev.slne.surf.surfapi.bukkit.api.nms.listener.packets.serverbound;

import org.jetbrains.annotations.ApiStatus.NonExtendable;

@NonExtendable
public non-sealed interface CommandSuggestionPacket extends NmsServerboundPacket{

  int completionId();

  String command();
}

File: surf-api-bukkit/surf-api-bukkit-api/src/main/kotlin/dev/slne/surf/surfapi/bukkit/api/nms/listener/packets/serverbound/CommandSuggestionPacket.kt
package dev.slne.surf.surfapi.bukkit.api.nms.listener.packets.serverbound

import dev.slne.surf.surfapi.bukkit.api.nms.NmsUseWithCaution

@NmsUseWithCaution
interface CommandSuggestionPacket : NmsServerboundPacket {
    val completionId: Int
    val command: String
}
